src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private K[] arrayKey = (K[]) new Object[MAX_ITEMS_NUMBER];
+    private V[] arrayValue = (V[]) new Object[MAX_ITEMS_NUMBER];
+    private int count = 0;
```

Comment:
```suggestion
    private int count;
```
0 is the default value of int